src/rohan/IdenticalStrings.java

```diff
@@ -0,0 +1,54 @@
+package rohan;
+
+import java.util.Scanner;
+
+/*
+ * Program 2: WAP to find two String are idential or not
+
+input 1: 
+String s1 = "technoCredits";
+String s2 = "teChnoCreditS";
+
+output 1: both string are not idential
+
+input 2:
+String s1 = "pune";
+String s2 = "pune";
+
+output 2: both string are idential
+ */
+
+public class IdenticalStrings {
+	
+	public static void main(String[] args) {
+		IdenticalStrings identicalStrings = new IdenticalStrings();
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter string 1: ");
+		String str1 = sc.next();
+		System.out.println("Enter string 2: ");
+		String str2 = sc.next();
+		identicalStrings.findIdenticalStrings(str1,str2);
+	}
+	
+	void findIdenticalStrings(String str1,String str2) {
+		boolean flag = true;
+		if(str1.length() == str2.length()) {
+			for(int index=0;index<str1.length();index++) {
+				if(str1.charAt(index) != str2.charAt(index)) {
+						flag = false;
+						break;
+				}
+				else {
+					if(str1.charAt(index) == str2.charAt(index) && Character.isLowerCase(str1.charAt(index))==Character.isLowerCase(str2.charAt(index)) || Character.isUpperCase(str1.charAt(index))==Character.isUpperCase(str2.charAt(index)))
```

Comment:
both condition are same if you observe if condition and ese me if condition.
you require only first if condition, rest code not require